test/integration/__init__.py

```diff
@@ -24,3 +24,7 @@
 # Workaround for the intermittent worker timeout errors
 # TODO: find and solve the root cause of this issue
 NUM_MODEL_SERVER_WORKERS = 2
+
+# EI is currently only supported in the following regions
+# regions were derived from https://aws.amazon.com/machine-learning/elastic-inference/pricing/
+EI_SUPPORTED_REGIONS = ['us-east-1', 'us-east-2', 'us-west-2', 'eu-west-1', 'ap-northeast-1', 'ap-northeast-2']
```

Comment:
can we read this from the Python SDK or something? just getting wary of having this hardcoded in too many places.

Reply:
It is in the init file in the test directory. I can't pull it out, however you do make a good point still.

Reply:
fair enough